### DISCUSSION TITLE
ERROR DRT-0085

### DISCUSSION THREAD

**Discussion by techstuds** [The-OpenROAD-Project/OpenROAD#3730] [Q&A]
I'm facing this error during global routing: 
`[ERROR DRT-0085] Valid access pattern combination not found for `
`pipe_mu3_mul_18_52_g4568`
`@@@ dead end inst`

What does this mean exactly and what could be causing it?

**Comment by maliberty** [comment]
The router is having some trouble figuring out how it can access all the pins of the instance.  Can you provide a test case?  If so please open an issue.

**Reply by techstuds** [reply]
I'm working on a suitable testcase and will upload it soon. Thanks!
